test/Tests/src/Data/Numbers_Spec.enso

```diff
@@ -226,6 +226,30 @@ spec =
         Test.specify "should define multiplication" <|
             2*3 . should_equal 6
 
+        Test.specify "should define modulo for integers and decimals" <|
+            5%3 . should_equal 2
+            5%3 . should_be_an Integer
+
+            5.0%3.0 . should_equal 2.0
+            5%3.0 . should_equal 2.0
+            5.0%3 . should_equal 2.0
+            5.0%3.0 . should_be_a Decimal
+            5%3.0 . should_be_a Decimal
+            5.0%3 . should_be_a Decimal
+
+            3.5%2 . should_equal 1.5
+            10.5%1.0 . should_equal 0.5
+
+            3%1 . should_equal 0
+            3%1 . should_be_an Integer
+
+            3.5%1 . should_equal 0.5
+
+            1%0 . should_fail_with Arithmetic_Error
+            1.0%0.0 . is_nan . should_be_true
+            1%0.0 . is_nan . should_be_true
+            1.0%0 . is_nan . should_be_true
+
```

Comment:
can you add a test with BigIntegers (e.g with `almost_max_long_times_three`) in addition to Longs and Doubles